os/include/assert.h

```diff
@@ -103,6 +103,13 @@ extern char assert_info_str[CONFIG_STDIO_BUFFER_SIZE];
 
 #ifdef CONFIG_DEBUG
 
+#define DEBUGASSERT_INFO(f, fmt, ...) \
+	{ if (!(f))	{ \
+			  snprintf(assert_info_str, CONFIG_STDIO_BUFFER_SIZE, fmt, ##__VA_ARGS__); \
+			  up_assert((const uint8_t *)__FILE__, (int)__LINE__); \
+			} \
+	}
+
```

Comment:
It looks something strange. How about below?
```
{ \
    if ( ) { \
        snprintf(); \
        up_assert(); \
    } \
}
```

Reply:
I will make the new macros with this indentation and change the indentation of the old macros in a separate commit in this PR itself